<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ThumbnailsDialog</class>
 <widget class="QDialog" name="ThumbnailsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>945</width>
    <height>514</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Thumbnails Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="1" column="0">
    <layout class="QGridLayout" name="left_gridLayout">
     <item row="2" column="6">
      <widget class="QComboBox" name="play_spd_comboBox">
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="2" column="3" colspan="2">
      <widget class="QPushButton" name="playpause_pushButton">
       <property name="text">
        <string>Play</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="stop_pushButton">
       <property name="text">
        <string>Stop</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QPushButton" name="select_pushButton">
       <property name="text">
        <string>Select Video</string>
       </property>
      </widget>
     </item>
     <item row="1" column="7">
      <widget class="QLabel" name="time_label">
       <property name="text">
        <string>00:00:00/00:00:00</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="7">
      <widget class="QSlider" name="progress_horizontalSlider">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="7">
      <widget class="QPushButton" name="snap_pushButton">
       <property name="text">
        <string>Snap</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="8">
      <widget class="QWidget" name="widget" native="true">
       <widget class="QLabel" name="video_label">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>651</width>
          <height>371</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 1px solid black</string>
        </property>
        <property name="text">
         <string>Drag Video File Here</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </widget>
     </item>
     <item row="2" column="5">
      <widget class="QComboBox" name="volume_comboBox">
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1" rowspan="2">
    <widget class="Line" name="separator_line">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(134, 134, 134);</string>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <layout class="QGridLayout" name="right_gridLayout">
     <item row="2" column="0" colspan="6">
      <widget class="QLabel" name="thumbs_process_label">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="4">
      <widget class="QPushButton" name="get_thumbs_pushButton">
       <property name="text">
        <string>Generate</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="6">
      <widget class="QListView" name="thumbs_listView"/>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <layout class="QGridLayout" name="topleft_gridLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="display_label_2">
       <property name="text">
        <string>缩略图生成目录:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="display_label_1">
       <property name="text">
        <string>当前视频位置:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>1</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="change_dir_pushButton">
       <property name="text">
        <string>更改</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QLineEdit" name="current_video_lineEdit">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>No Video Selected.</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="thumbs_dir_lineEdit">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="2">
    <layout class="QGridLayout" name="topright_gridLayout">
     <item row="1" column="3">
      <widget class="QSpinBox" name="column_spinBox"/>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QPushButton" name="del_thumb_pushButton">
       <property name="text">
        <string>删除选中的</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2" colspan="2">
      <widget class="QPushButton" name="clear_thumbs_pushButton">
       <property name="text">
        <string>清空</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="display_label_4">
       <property name="text">
        <string>列：</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="display_label_3">
       <property name="text">
        <string>行：</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="row_spinBox"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
